router/gossip.go

```diff
@@ -68,6 +68,7 @@ func (sender *GossipSender) run() {
 			case pending := <-sender.cell:
 				if pending != nil {
 					sender.send(pending)
+					ch <- true
```

Comment:
we don't actually need the nil check here.

Also, I reckon we should send true or false, and *not* close the channel.